### DISCUSSION TITLE
2024/06/30/20240630_updatelog/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#24] [General]
# 2024/06/30/20240630_updatelog/

如题。

https://wz.anoms.top/2024/06/30/20240630_updatelog/

**Comment by Hazelandinstant** [comment]
我来讲讲最近遇到的一件“关心和爱”的事吧：周末自己去电影博物馆看电影，选了IMAX屏的纪录片（据说这个屏是最大的，至于在哪个level比较我毫不在意），但是看完却感觉浑身不适。于是决定买个冰淇凌给自己吃，一个人做决定很容易，选定口味直接付钱就可以，整个过程一气呵成，没什么犹豫。
当我准备拿冰淇凌时候，后面还有一个父亲和女儿在排队，大概四五岁的小女孩在选择自己喜欢的冰淇凌，但是老父亲却并不愿意为此买单，一直强调“旁边这个更健康”，而小女孩则是既期待又胆怯地和她的父亲争论“我就是想要这个”。
其实有时候现做的冰淇凌和冰箱里包装好的雪糕在口味上并没什么差别，甚至好口味的雪糕更胜一筹，因为口味稳定熟悉，而现做的冰淇凌更像是开盲盒。我并不是想要争辩到底是冰淇凌还是雪糕更好吃更划算，而是想说，当我为自己买单毫不犹豫的时候，我感觉自己就在“爱自己”，不用听取别人的意见——因为这两者本来就是各自的偏爱，没什么争论的必要；如果喜欢做这件事而自己又有能力去做（也没有伤害别人和自己），那毫不犹豫向前迈进大概也是爱自己的一种方式。
我最近感受到很多人对我的关爱，我也深深感动。但是整个六月份的所有关于爱的事情中，这件是让我感受最真实的，非常真实的对自己的爱。
以前我听到类似的“首先要自己爱自己”这样的话，总觉得很扯。但现在发现只有自己才能精准发现自己的需求，也只有自己对自己的爱最恰当。从父亲的角度考虑也没错，孩子的想法也没错；父亲的想法也是爱，但可能不是“我”这个孩子需要的爱。

**Reply by cyclinbox** [reply]
感觉佳桢的文笔像是在写散文，一些细致的观察和感受如流水般娓娓道来 :-)
关于“冰激凌”的故事，父亲和女儿的一些争论某种程度上代表了两代人的利益冲突；而站在孩子的立场上，被大人硬塞的“喜好”总归是不快乐的。而“感觉自己就在‘爱自己’，不用听取别人的意见”这一细致的感受，确实也是会让人感到幸福和充实的呀！前段时间的六一，一些公众号在站在年轻人的视角上庆祝大人的六一——“终于成为了大人，可以自己决定自己的生活”，细想一下也是这样。
做自己喜欢的事情，勇敢的向前迈进，用这种方法自己爱自己——这种来自于我们自身的爱，可以带来许多力量。愿我们都能在充满自信和爱的环境中自由生长！
以及，夏天快乐！（不知道如何结尾了，就送一句祝福吧~

**Comment by THUDBT** [comment]
踩踩

**Reply by cyclinbox** [reply]
谢谢awa~

**Reply by Hazelandinstant** [reply]
让我猜猜： TsingHuaDragonBoatTeam
